index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+
+const knexConfig = {
```

Comment:
This isn't needed here. you simply need to import in your knex file with the same info.